docs/source/cloud-setup/policy.rst

```diff
@@ -339,10 +335,12 @@ Use local GCP credentials for all tasks
     :pyobject: UseLocalGcpCredentialsPolicy
     :caption: `UseLocalGcpCredentialsPolicy <https://github.com/skypilot-org/skypilot/blob/master/examples/admin_policy/example_policy/example_policy/client_policy.py>`_
 
+Specify the following config in :ref:`the SkyPilot config <config-yaml>` at the client:
+
 .. literalinclude:: ../../../examples/admin_policy/use_local_gcp_credentials.yaml
     :language: yaml
     :caption: `Config YAML for using UseLocalGcpCredentialsPolicy <https://github.com/skypilot-org/skypilot/blob/master/examples/admin_policy/use_local_gcp_credentials.yaml>`_
 
-.. note::
+Then specify the policy at the server with the same config, or call this policy in the :ref:`RESTful policy server <host-admin-policy-as-server>`.
 
-    This policy only take effects when applied at :ref:`client-side <client-side-admin-policy>`. Use this policy at the :ref:`server-side <server-side-admin-policy>` will be a no-op.
+In the above example, the policy has to access the user's local environment to discover the user's GCP credentials and upload the credentials to the cluster by modifying the ``file_mounts`` field. So the key functionality of this policy must be executed as a client-side policy. In addition, when this policy is called at server-side, it will check whether a policy of the expected version has been applied at client-side and rejects the request if not. So the client-side application of the policy can be enforced on the server for non-malicious users.
```

Comment:
```suggestion
The policy above can access the user's local environment to discover the user's GCP credentials and upload the credentials to the cluster by modifying the ``file_mounts`` field. Note that the key functionality of this policy is executed as a client-side policy to discover the local credentials.

When this policy is called at server-side, it will check whether a policy of the expected version has been applied at client-side and rejects the request if not.
```

Reply:
This is much clearer, thanks!